.github/workflows/deploy-npm.template.yml

```diff
@@ -40,11 +40,11 @@ jobs:
 
       - name: Set version
         working-directory: ${{ inputs.working-directory}}
-        run: yarn version ${{ steps.parse_version.outputs.VERSION }}
+        run: npm version ${{ steps.parse_version.outputs.VERSION }} --allow-same-version --no-commit-hooks --no-git-tag-version
```

Comment:
Does this flag mean we can publish new code to the same NPM version? E.g. "update" v1.03 for example?

Reply:
So this is strictly for setting the `package.json` version field, without the `--allow-same-version` flag if the `package.json` had a version of `1.2.3` and then we ran the `npm version 1.2.3` command, it would error out saying "version is the same". IMO the expected behavior there is it should be a no-op, so by passing that flag it allows to create a GH release tagged as the same version that already exists in the `package.json`.

Reply:
Got it. We're still intending NPM version to be immutable. 👍